### DISCUSSION TITLE
How stable is this?

### DISCUSSION THREAD

**Discussion by rangertaha** [noneback/go-taskflow#41] [Q&A] [answered]
Are you planning on making breaking changes anytime soon? I am looking for a better task executor for https://github.com/rangertaha/urlinsane and would love to try this.

**Comment by noneback** [comment, marked as answer]
Hey. Recently, there have been no breaking changes for the existing exported APIs(v0.1.2). We may add some new APIs for better user experience or utility, and it should not affect the existing API. But still, it is in the early stage of development, so it is not suggested to use it in **production** until V1.0.0 is released. We will inform users in advance if we are making any breaking changes.

And I would appreciate it if you would like to try this. Feel free to reach out if you have any questions or need any help.

**Reply by rangertaha** [reply]
Thanks
